packages/oui-textarea/src/textarea.html

```diff
@@ -0,0 +1,22 @@
+<div class="oui-textarea"
```

Comment:
We can avoid this useless DIV by adding all those classes on the root element, to generate only:
```
<oui-textarea class="oui-textarea">
    <textarea></textarea>
</oui-textarea>
```
The `disabled` and `readonly` part could be changed with $onChange if necessary

Reply:
So bloated code. I really don't like this old broken deprecated UI framework...

Reply:
I also thought about it and when I come to wonder if I should use the component tag... I try to know if this could could be used with `ng-repeat`. If no, I don't make the effort.